kafka/README.md

```diff
@@ -0,0 +1,37 @@
+# Kafka SpecMesh extension
+
+Extends SpecMesh with the concept of public, protected and public topics.
+
+## Authorisation
+
+## Topic authorisation
+
+The provisioner will set ACLs to enforce the following topic authorisation:
+
+### Public topics:
+
+Only the domain itself can `WRITE` to public topics; Any domain can `READ` from them.
+
+### Protected topics:
+
+Only the domain itself can `WRITE` to protected topics; Any domain specifically tagged in the spec can `READ` from them.
```

Comment:
similarly - some kind of example? wdyt?

Reply:
Agreed, but outside the scope of this PR.  I'm just wanting to document what I did in this and the last PR around ACLs.